app/views/site_channels/verification_public_file.html.slim

```diff
@@ -4,6 +4,9 @@
     publisher: current_publisher \
   }
 
+script type="text/html" id="verification_failed_modal_wrapper" data-open-verification-failed-modal="#{current_channel.verification_failed?}"
```

Comment:
This script (here and on all of the verification pages) will load the modal if the channel's current `verification_status` has failed.  This has the unfortunate side effect of displaying the modal immediately after a publisher selects a new verification method.  I think ideally we show the failed verification modal only if the the failed attempt uses the method of the page the publisher is currently on.  I'll think about how to do this...